fab/environments.yml

```diff
@@ -107,6 +111,8 @@ production:
         concurrency: 200
       ucr_queue:
         concurrency: 4
+      ucr_indicator_queue:
+        concurrency: 10
```

Comment:
is 10 needed? that might be a lot

also i think it would be good to put this on celery2